content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -40,30 +40,40 @@ It's good practice to structure your files well, so we'll expect you to have a s
 └──.gitignore 
 ```
 
+## Dealing with missing values in a dataset
 
-## Questions
+Before we dive into the tasks of this project, let's talk a bit about missing values. For a variety of reasons, the data we work with could have missing or invalid data. Depending on the kind of data we are working with and the extent of the data missing, we typically deal with missing data either by dropping it (for example in the case of entirely/nearly entirely missing columns or rows) or by replacing them with a suitable values.
+
+Later in this program you'll be learning how to train machine learning models using various kinds of data. When doing so, missing values cannot simply be ignored. Further, when it comes to datasets with many columns (also known as features), it is bad practice to drop an entire row for a few missing values. 
+
+Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+
+
+## Instructions
 
 The [data set](data.csv) consists of health and demographic data for the period 2014-2015, obtained from [Global Health Observatory Data Repository](http://apps.who.int/gho/data/node.main). Here is some [metadata](data-info.txt) that may be useful.
 
-1. Are there any missing values in the data set? You can deal with missing values by either dropping them or filling them with some value (in a variety of ways) or ignoring them. What do you think is best in this case? And why? Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+1. Load the dataset into a pandas DataFrame and determine how many missing values there are per feature. 
+
+2. Address any missing values in the dataset and lay out your reasoning for your chosen method. Given the kind of data we have in this dataset, as well as the *regional* analysis you will be tasked with performing below, it should not simply be replacing a missing value with the mean/median of the whole column. 
 
-2. Are there any other problems with the data? If so, fix them.
+3. Are there any other problems with the data? If so, fix them. At this stage ensure that your dataset is in a DataFrame named: `health_and_demographics_df`
```

Comment:
Can't we use more direct language. Like "store the final datadrame in a variable named ..." or "name your dataframe ..."